src/main/java/net/coderbot/iris/gl/blending/BlendModeStorage.java

```diff
@@ -32,6 +34,10 @@ public static void overrideBlend(BlendMode override) {
 		}
 
 		blendLocked = true;
+
+		if (blendFuncListener != null) {
+			blendFuncListener.run();
```

Comment:
This isn't correct. This needs to be run at the end of `GlStateManager`'s blendFunc or blendFuncSeparate functions, not here - the blendFunc uniform isn't just for blend mode overrides.

Reply:
Fixed.